app/src/components/Chats/ChatPreview.tsx

```diff
@@ -66,32 +67,27 @@ export default function ChatPreview({
   const firstMessage = sortedMessages ? sortedMessages[0] : null
   const lastMessage = sortedMessages && sortedMessages.length > 1 ? sortedMessages.at(-1) : null
   const repliesCount = sortedMessages ? sortedMessages.length - 2 : 0
-  const showName =
-    // const messageUser = lastMessage ? chatUsers[lastMessage.authorId] : null
 
-    useEffect(() => {
-      setUnreadMessages(getUnreadMessagesCount(chat.id))
-      // console.log(chatUsers[firstMessage.authorId].name)
-    }, [unreadChats])
+  useEffect(() => {
+    setUnreadMessages(getUnreadMessagesCount(chat.id))
+  }, [unreadChats])
 
-  const disableChat = unreadMessages === 0
+  const isChatDisabled = unreadMessages === 0
 
   return (
-    // <div
-    //   className='w-full flex border-border-color hover:cursor-pointer hover:bg-gray-100 pr-7  py-3'
-    //   onClick={onClick}
-    // >
     <div className='flex w-full'>
       <div
-        className={`w-2 h-full border-r-2 border-border-color ${
-          disableChat ? 'bg-[#EFEFEF]' : 'bg-[#DBF7CA]'
-        } `}
+        className={clsx(
+          'w-2 h-full border-r-2 border-border-color',
+          isChatDisabled ? 'bg-[#EFEFEF]' : 'bg-[#DBF7CA]',
+        )}
       ></div>
       <div
-        className={`w-full flex border-border-color hover:cursor-pointer hover:bg-gray-100 pr-7  py-3 ${
-          disableChat ? 'pointer-events-none opacity-50' : ''
-        }`}
-        onClick={disableChat ? undefined : onClick}
+        className={clsx(
+          'w-full flex border-border-color hover:cursor-pointer hover:bg-gray-100 pr-7 py-3',
+          isChatDisabled ? 'pointer-events-none opacity-50' : '',
+        )}
+        onClick={isChatDisabled && onClick}
```

Comment:
!isChatDisabled* my bad 🫠

Reply:
sorry eugeek I did not understand what to do in this part

Reply:
isChatDisabled - when chat is disabled and onClick should not work.
So !isChatDisabled - chat is not disabled and onClick should work

Reply:
if chat is disabled how onclick will work?